### PR TITLE
Bump `erblint-github` and make sure it is actually running.

### PR DESCRIPTION
**Pull request by khiga8**
_Authors: Please fill out this form carefully and completely._

_Reviewers: By approving this Pull Request you are approving the code change, as well as its deployment and mitigation plans._
_Please read this description carefully. If you feel there is anything unclear or missing, please ask for updates._

### What are you trying to accomplish?

* Bump `erblint-github` to latest
* Enable latest rules
* Make sure that the rules are actually running by creating `.erb-linters/erblint-github.rb` and requiring the rules from the gem.

### Integration
No

#### List the issues that this change affects.
https://github.com/github/accessibility/issues/3766

#### Risk Assessment
  

- [x] **Low risk** the change is small, highly observable, and easily rolled back.
- [ ] **Medium risk** changes that are isolated, reduced in scope or could impact few users. The change will not impact library availability.
- [ ] **High risk** changes are those that could impact customers and SLOs, low or no test coverage, low observability, or slow to rollback.

### What approach did you choose and why?

I bumped the gem because it has new rules.

### Anything you want to highlight for special attention from reviewers?
No

### Merge checklist

- [ ] Added/updated tests
- [ ] Added/updated documentation
- [ ] Added/updated previews (Lookbook)
- [ ] Tested in Chrome
- [ ] Tested in Firefox
- [ ] Tested in Safari
- [ ] Tested in Edge

Take a look at the [What we look for in reviews](https://github.com/primer/react/blob/main/contributor-docs/CONTRIBUTING.md#what-we-look-for-in-reviews) section of the contributing guidelines for more information on how we review PRs.

### CODE REVIEW

File: .erb-linters/erblint-github.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+require "erblint-github/linters"
```
Comment (khiga8) [inline .erb-linters/erblint-github.rb:1]: I don't believe there's currently a better way to inherit configs or rules from another gem in ERBLint

Related issue: https://github.com/Shopify/erb-lint/issues/317.

**Review by jonrohan — APPROVED** [review]

**Comment by khiga8** [comment]
moving to draft.

going to deprecate counter code from erblint-github which seems to interfere here.

File: app/components/primer/alpha/octicon_symbols.html.erb [hunk 2]
```diff
@@ -1,3 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" hidden>
+<svg xmlns="http://www.w3.org/2000/svg" hidden> <%# erblint:disable GitHub::Accessibility::SvgHasAccessibleText %>
```
Comment (khiga8) [inline app/components/primer/alpha/octicon_symbols.html.erb:1]: This should be addressed.

**Comment by khiga8** [comment]
This is ready for another review.

**Comment by khiga8** [comment]
Could we add `erblint-github` to dependabot to make sure we stay up to date?

**Pull request merged**